src/core/tour/TourManager.ttslua

```diff
@@ -84,10 +98,21 @@ do
     hook.setPositionSmooth(CAMERA_HOME, false, false)
     local delay = 0.5
     local cardIndex = tourState[playerColor].currentCardIndex
+    local lookPos
     if TOUR_SCRIPT[cardIndex].showObj ~= nil then
+      local lookAtObj = getObjectFromGUID(TOUR_SCRIPT[cardIndex].showObj)
+      lookPos = lookAtObj.getPosition()
+      lookPos.y = TOUR_SCRIPT[cardIndex].distanceFromObj or 0
+      -- Since camera isn't directly above the hook, changing the Y affects the visual position of
+      -- whatever object we're trying to look at.  This is an approximation, but close enough to
+      -- keep the object more centered
+      lookPos.x = lookPos.x - lookPos.y / 2
+    elseif TOUR_SCRIPT[cardIndex].showPos ~= nil then
+      lookPos = TOUR_SCRIPT[cardIndex].showPos
+    end
+    if lookPos ~= nil then
       Wait.time(function()
-        local lookAtObj = getObjectFromGUID(TOUR_SCRIPT[cardIndex].showObj)
-        hook.setPositionSmooth(lookAtObj.getPosition(), false, false)
+        hook.setPositionSmooth(lookPos, false, false)
       end, delay)
       delay = delay + 0.5
```

Comment:
Does increasing the delay actually do anything since it gets set to 0.5 before?

Reply:
Yes - this is the delay until the next card is shown (used below) and needs to be different based on whether the camera is centering, or is moving to a new spot (each of which takes half a second)